api/src/controllers/user.js

```diff
@@ -85,124 +86,142 @@ exports.list = (req, res) => {
 	}
 };
 
-exports.delete = (req, res) => {
+exports.delete = async (req, res) => {
```

Comment:
@dwightgunning you have to wrap the routes for these controllers with `wrapAsync` or bad stuff will happens

Reply:
👍